### PR TITLE
#83 Long Booking numbers get cut by ellipsis

### PR DESCRIPTION
**Pull request by mathvaillant**

### CODE REVIEW

**Comment by mathvaillant** [comment]
[Long Booking numbers get cut by ellipsis](https://trello.com/c/9aHZ3pAC/83-long-booking-numbers-get-cut-by-ellipsis)

**Review by joaoqueiroz95 — COMMENTED** [review]
I do not understand how these changes do not make it disappear

**Comment by mathvaillant** [comment]
> I do not understand how these changes do not make it disappear

What are you talking about? This ticket is only related to the text too big on the error toaster

File: src/config/toast/toast.tsx [hunk 1]
```diff
@@ -51,12 +51,16 @@ const styles = {
...
+    text2: {},
   }),
   error: StyleSheet.create({
     container: {
       backgroundColor: theme.colors.symbols.red,
     },
     text1: {},
+    text2: {
+      fontSize: 14,
+    },
```
Comment (joel-gouveia) [inline src/config/toast/toast.tsx:63]: It font size the solution? wouldn't it be better to have a third line of text just for the number?

Comment (mathvaillant) [reply src/config/toast/toast.tsx:63]: Font size solves the issue, should I still go on and create a custom toaster? to have more than 2 texts we're gonna have to do that.

**Comment by joaoqueiroz95** [comment]
> > I do not understand how these changes do not make it disappear
> 
> What are you talking about? This ticket is only related to the text too big on the error toaster

Sorry, thought this was the ticket related to the field being hidden in summary when the label is too long

**Review by joaoqueiroz95 — APPROVED** [review]

**Pull request merged**